### DISCUSSION TITLE
collapsible with direction icons

### DISCUSSION THREAD

**Discussion by subdesign** [livewire/livewire#5364] [1. Help]
I have a similar slidedown/collapse like here: https://alpinejs.dev/plugins/collapse 
It has a form in it and for the button I use chevron-up and chevron-down svg icons for visually toggle them. I change the hidden classes like so:
```html
<div x-data="{ expanded: false }">
    <button @click="expanded = ! expanded">Toggle Content
        <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6 chevron-down" :class="expanded ? '' : 'hidden'"><path stroke-linecap="round" stroke-linejoin="round" d="M19.5 8.25l-7.5 7.5-7.5-7.5" /></svg>
        <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6 chevron-up" :class="expanded ? 'hidden' : ''"><path stroke-linecap="round" stroke-linejoin="round" d="M4.5 15.75l7.5-7.5 7.5 7.5" /></svg>
    </button>
 
    <p x-show="expanded" x-collapse>
        <form>
        ...
        </form>
    </p>
</div>
```
It works until I submit the form in the collapsible, or only click on a checkbox, or on an input field, because it adds `hidden` class to both of the svgs. 
After that if I click on the toggle button again, it removes/adds the `hidden` class only from the second svg.

removes hidden only on second svg:
<img width="261" alt="Screenshot 2022-11-15 at 11 50 30" src="https://user-images.githubusercontent.com/691695/201901708-4ee77c10-1cfc-4861-b974-8424297c984e.png">
adds hidden only on second svg.
<img width="264" alt="Screenshot 2022-11-15 at 11 50 42" src="https://user-images.githubusercontent.com/691695/201901752-41f29163-7ef3-43ee-83d9-bc633b8bc8d1.png">

Not clear if the `expanded` variable is available in the div, why not working on the first svg? Maybe when Livewire communicates with the backend it forgets expanded's state?

**Comment by subdesign** [comment]
JQuery was loaded first, so I changed the order, put `<livewire:scripts />` first, and below the script tag of JQuery. 
Now everything works. Also "Illegal invocation" js error disappeared.

**Reply by joshhanley** [reply]
@subdesign yep, that's correct, Livewire scripts should be loaded first and then everything else, and then Alpine last.

Hope this helps!
